Updated readme for adding words to wordlist (#20)
openSUSE/suse-documentation-dicts: README.md

> ## Spellcheck dictionaries for SUSE documentation

Reviewer: ```suggestion
## Spell-check dictionaries for SUSE documentation
```

This dictionary improves the spellchecking experience for SUSE and openSUSE
documentation by providing an extra word list that can be used in addition
to the regular en_US dictionary.

## Installation

You can either install the dictionaries from your distribution packages or build them manually.

### Installing the package version of the dictionaries

* For openSUSE Leap and Tumbleweed users, the easiest option is installing the
  packaged version of the dictionaries from
  https://build.opensuse.org/package/show/Documentation:Tools/suse-documentation-dicts-en.
* For Debian and Ubuntu users, find the package at
  https://build.opensuse.org/package/show/home:tbazant/suse-documentation-dicts

> ### Building the Dictionaries

Reviewer: ```suggestion
### Building the dictionaries
```

```

> ## Using the Dictionaries

Reviewer: ```suggestion
## Using the dictionaries
```

```

### Hunspell

```
hunspell -H -i utf-8 -d en_US,en_US-suse-doc FILE
```

> ### Vim Word List & Vim

Reviewer: ```suggestion
### Vim word list & Vim
```

> ### Adding Words

Reviewer: ```suggestion
### Adding words
```

> The doctionaries include a word list of correct words that are not part of the
> standard English dictionaries. The word list is being extended as the writers
> find new words that are not yet part of the word list but still are correct. If
> you want to contribute to improving the spell-checking experience, collect such words that the spell checker marks as incorrect but you consider them correct.

Reviewer: ```suggestion
The dictionaries include a word list of correct words that are not part of the
standard English dictionaries. The list of acceptable words is updated as 
writers discover words that are correct but not yet included. To improve 
spell checking, please collect words that the spell checker flags as incorrect 
but you believe are correct.
```

> TIP: If you are using VSCode, you probably have such file already. Check if the
> `SUSE-adWords-dict.dic` file exists. If yes, VSCode lets you add the suspect
> word to this dictionary by clicking the corresponding small yellow bulb next to
> the affected line.

Reviewer: ```suggestion
TIP: If you are using VSCode, you probably have such a file already. Check if the
`SUSE-adWords-dict.dic` file exists. If yes, VSCode lets you add the suspect
word to this dictionary by clicking the corresponding small yellow bulb next to
the affected line.
```

> Once you collect a certain amount of such words, follow these steps:

Reviewer: ```suggestion
Once you collect a certain amount of new words, follow these steps:
```

1. Clone the `https://github.com/openSUSE/suse-documentation-dicts` repository
   if you have not already done so.
  ```
  git clone git@github.com:openSUSE/suse-documentation-dicts.git
  ```
2. Create a branch for your changes, for example:
  ```
  git branch tbazant-adding-custom-words
  ```
3. Checkout to this branch, manually or, for example, in VSCode source control pane.
  ```
  git checkout tbazant-adding-custom-words
  ```
4. Edit the `suse_wordlist.txt` file and add all your words to the end of the
  existing list. Avoid duplicating regular dictionary words and words that are
  already in the list.

    TIP: The word list format supports defining suffixes to avoid word
    repetitions. Suffixes are appended to the regular entry this way:

    ```
    entry +suffix +y/iesuffix
    ```

    This produces the following three words in the output word list:

    ```
    entry
    entrysuffix
    entriesuffix
    ```

    * There must be exactly one space before each suffix definition.

>     * Suffix definition start with a `+` character.

Reviewer: ```suggestion
    * The suffix definition starts with a `+` character.
```

    * There are two forms of suffixes:
      * Without replacement, like `+suffix`: the characters `suffix` are appended
        to the entry.
      * With replacement, like `y/iesuffix`: the character `y` is removed from the
    end of the entry and `iesuffix` is appended.

  5. Sort and validate the enhanced word list file:
    ```
    make sortvalid
    ```

>   6. Commit the changes to your branch.

Reviewer: ```suggestion
  6. Commit the changes to your branch:
```
